pos_remove_pos_category/product.py

```diff
@@ -44,12 +44,23 @@ def write(self, vals):
 class ProductCategory(models.Model):
     _inherit = 'product.category'
 
+    image = fields.Binary(help='Show Image Category in Form View')
+    image_medium = fields.Binary(help='Show image category button in POS')
```

Comment:
Hello , I tried , but did not have the desired results , so I chose to use @api.constrains